src/components/Collapse.js

```
@@ -0,0 +1,27 @@
+        onClick={() => setIsOpen(!isOpen)}
+        style={{
+          background: 'none',
+          border: 'none',
+          fontSize: '16px',
+          fontWeight: 'bold',
+          cursor: 'pointer',
+          display: 'flex',
+          alignItems: 'center'
+        }}
+      >
+        {isOpen ? '▼' : '▶'} <span dangerouslySetInnerHTML={{ __html: summary }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)